dev-tools/packaging/package_test.go

```diff
@@ -268,6 +275,35 @@ func checkSystemdUnitPermissions(t *testing.T, p *packageFile) {
 	})
 }
 
+// Verify that modules.d folder is present and has module files in
+func checkModulesPresent(t *testing.T, prefix string, p *packageFile) {
+	minExpectedModules := 4
```

Comment:
In the future we could make this a param that can also be passed to make sure we didn't miss a module.